internal/openshift/controller.go

```diff
@@ -31,7 +32,8 @@ type Controller interface {
 	GetUser(ns string) model.User
 }
 
-type OpenShiftController struct {
+// ControllerImpl watches OpenShift cluser for changes and implements Controller
+type ControllerImpl struct {
```

Comment:
I don't think you answered earlier. Why 'Impl'. I think it is not needed. It could imo just be 'controller'. Notice, lower case 'c'. We don't need to export this type. All the code should be using the interface.

Reply:
Yes, my bad. I know you had mentioned this earlier, but couldn't recall when I was coding. `controller` would definitely work. Let me rename this to `controller`

Reply:
there is a variable named `controller`. Let's keep `ControllerImpl`, it is more descriptive. `Controller` and `controller` and confusing as well.